src/Microsoft.Data.Analysis/DataFrameColumnWindow.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Data.Analysis
+{
+    /// <summary>
+    /// A window object that can be used for expanding and rolling computations
```

Comment:
What does `expanding` computation mean? Do we have that ability?

Reply:
Expanding is another type of window computation. A rolling window slides over the values with the window size constant. An expanding window expands as it slides over the values. So at an index `i`, it would include everything from 0..i. We don't support it yet. You're right, I should change the doc to be more clear.